### ISSUE TITLE
Cleanup landing page tab UI

### ISSUE THREAD

**Issue by jasonLaster** [devtools-html/devtools-core#72]
* The landing page tabs could line up nicely with the left sidebar tabs if we update the vertical heights.
* also we can cleanup to focus border to be less obnoxious

Should be easy to do.

![screen shot 2017-01-06 at 12 05 06 pm](https://cloud.githubusercontent.com/assets/254562/21725924/6cad9ca2-d408-11e6-879b-3b24ac54a0a6.png)

**Comment by nchevobbe** [comment]
I see a great opportunity to use CSS grids here :)

**Comment by jasonLaster** [comment]
Oh nice. Would love to see that!

Let's get that in there

**Comment by nchevobbe** [comment]
![screen shot 2017-01-12 at 13 56 25](https://cloud.githubusercontent.com/assets/578107/21890655/ff190228-d8ce-11e6-8f72-d6995aa58d4f.png)

Using grid is nice here :) But I'm not pleased with the alignement of the "Firefox" element with the tab item on the right. I don't see an obvious solution to align those since the tab on the right has a variable height (if the title or the URL is long enough, they can be multiline)

I'll try to iterate a bit on the page

**Comment by jasonLaster** [comment]
Hmm, one solution would be to right align the URL on the same line as the title.

This way with a smaller font size, we should be fine.
Also if you want you can do two lines for the URL: 
```
host
url part (truncated potentioally)
```

**Comment by nchevobbe** [comment]
Yeah, I thought about it, the problem is that the title also can be multiline.
This can be cropped too, I wonder how it would look

**Comment by nchevobbe** [comment]
But yeah, one thing I'll try is too have a fixed height (the same one) for the tab element and the connection one, and do some text-overflow on both the title and the URL

**Comment by jasonLaster** [comment]
sounds good.

Remember, this is nice... but not *that* important :)

**Comment by AnshulMalik** [comment]
Do we still want this?
